fractal_server/app/models/workflow.py

```diff
@@ -68,7 +68,7 @@ def arguments(self):
         parallelization_level)
         """
         out = self.task.default_args.copy()
-        out.update(self.args)
+        out.update(self.args or {})
         popget(out, "parallelization_level")
         popget(out, "executor")
```

Comment:
Reminder: `executor` is now part of `meta` https://github.com/fractal-analytics-platform/fractal-server/issues/216